unstructured/documents/html.py

```diff
@@ -205,17 +201,7 @@ def _parse_pages_from_element_tree(self) -> list[Page]:
         page = Page(number=page_number)
 
         for article in self._articles:
-            for element in self._parse_article_to_elements(article):
-                # -- `<hr/>` element produces a `PageBreak` element, but is not added to elements --
-                if element == PageBreak(""):
-                    # -- ignore leading page-break and second and later consecutive page-breaks --
-                    if not page.elements:
-                        continue
-                    pages.append(page)
-                    page_number += 1
-                    page = Page(number=page_number)
-                else:
-                    page.elements.append(element)
+            page.elements = list(self._parse_article_to_elements(article))
```

Comment:
Looking at the second commit.
In the first commit, you added to the `_parse_pages_from_element_tree` docstring that "A horizontal-rule (`<hr/>`) element also triggers a page break", but you're removing that functionality here (removing lines 186-188)?

Reply:
*noting for myself to check that the docstring is updated again in a later commit.